helm/g2p-sandbox/values.yaml

```diff
@@ -568,6 +568,32 @@ ph-ee-engine:
       annotations:
         deployTime: "{{ .Values.deployTime }}"
 
+  connector:
+    enabled: true
+    image: docker.io/openmf/ph-ee-connector-bulk:latest
+    hostname: "connector.sandbox.mifos.io"
+    ingress:
+      enabled: true
+      annotations:
+        nginx.ingress.kubernetes.io/access-control-allow-origin: 'true'
+        nginx.ingress.kubernetes.io/cors-allow-methods: PUT, GET, POST, OPTIONS, DELETE
+        nginx.ingress.kubernetes.io/cors-allow-origin: '*'
+        nginx.ingress.kubernetes.io/enable-cors: 'true'
+      tls:
+        - secretName: sandbox-secret
+      hosts:
+        - host: connector.sandbox.mifos.io
```

Comment:
compare with ph-ee-engine values.yaml and remove duplicate values from here.
ingress should be disabled in g2p sandbox

Reply:
This is removed